frontend/server/src/Controllers/Problem.php

```diff
@@ -1721,7 +1721,7 @@ private static function getValidProblemAndProblemset(\OmegaUp\Request $r): array
      * Get the extra problem details with all the validations
      * @return null|array{statement: array{language: string, images: array<string, string>, markdown: string}, settings: array{cases: array<string, mixed>, limits: array{TimeLimit: string, OverallWallTimeLimit: string, MemoryLimit: int}, validator: mixed}, preferred_language?: string, problemsetter?: array{username: string, name: string, creation_date: int}, version: string, commit: string, title: string, alias: string, input_limit: int, visits: int, submissions: int, accepted: int, difficulty: null|float, creation_date: int, source?: string, order: string, points: null|float, visibility: int, languages: string[], email_clarifications: bool, runs?: array{guid: string, language: string, status: string, verdict: string, runtime: int, penalty: int, memory: int, score: float, contest_score: float, time: int, submit_delay: int, alias: string, username: string}[], admin?: bool, solvers?: array{username: string, language: string, runtime: float, memory: float, time: int}[], points: float, score: float}
      */
-    private static function getProblemDetails(
+    public static function getProblemDetails(
```

Comment:
```suggestion
    private static function getProblemDetails(
```

ve el comentario de arriba.

Reply:
Listo, se regresa como estaba.